### DISCUSSION TITLE
How can I save the pdf to an image as landscape instead of portrait?

### DISCUSSION THREAD

**Discussion by aatishdumps** [pypdfium2-team/pypdfium2#198] [Q&A]
I am using this code currently. I want the image in landscape orientation.

```python
imagepath = os.path.join(folder, name + '.jpg')
pdf = pdfium.PdfDocument(filepath)
page = pdf[0]
pil_image = page.render().to_pil()
pil_image.save(imagepath)
```

(edit by @mara004: syntax highlighting)

**Comment by mara004** [comment]
I suppose you're looking for the `rotation` parameter of `page.render()`?

**Reply by aatishdumps** [reply]
rotation rotates the content too.. i am looking for page orientation

**Reply by mara004** [reply]
What exactly do you intend to do? Resize the page or something?
In that case, you could [capture the page as XObject](https://pypdfium2.readthedocs.io/en/stable/python_api.html#pypdfium2._helpers.document.PdfDocument.page_as_xobject), place it on a new page, and then render. (Perhaps just modifying the existing page's mediabox/cropbox might also work.)

Apart from that, working with the image instead of the pdf may be an option, but that's about PIL usage and not pypdfium2.

If that's not what you mean, please be more specific, and ideally include an example.
